README.md

```diff
@@ -23,7 +23,39 @@ If you don't have an account, you can create one [here](https://www.livechatinc.
 To import LiveChat for React Native, run the following command:
 
 ```javascript
-npm install react-native-livechat --save
+npm install react-native-webview react-native-livechat --save
+```
+
+### Supported LiveChat features
+
+- chatting
+- sneak-peek
+- rich messages: single cards with images, title, subtitle and quick replies
+- agent's attachments
+- system messages
+
+### Unsupported LiveChat features
+
+- pre-chat and post-chat forms
+- rating
+- transcript
+- ticket forms
+- queues
+- translations
+- card masking
+- chat boosters
+
+If you want to use unsupported features in your React Native app, read about the Alternative React Native installation method.
+
+### Alternative React Native installation method
+
+This project implements basic LiveChat features using React Native technology. If you need to use more advanced LiveChat features, you can use Webview to embed web LiveChat widget using a direct chat link.
+
+To do it, use `react-native-webview` library and add `Webview` component with your licenses direct chat link, replacing `LICENSE_ID` string with you LiveChat's license id. 
+
+```javascript
+<WebView source={{uri: 'https://secure.livechatinc.com/licence/LICENSE_ID/v2/open_chat.cgi'}} />
```

Comment:
It is common practise to wrap placeholders with `<>` to indicate that it is acctually only a placeholder not real part of the URL
```suggestion
<WebView source={{uri: 'https://secure.livechatinc.com/licence/<LICENSE_ID>/v2/open_chat.cgi'}} />
```